src/ws/session-utils.c

```diff
@@ -530,7 +527,6 @@ static const char *env_names[] = {
   "G_MESSAGES_DEBUG",
   "G_SLICE",
   "PATH",
-  "COCKPIT_REMOTE_PEER",
```

Comment:
We might want to keep this as well.  It's still documented, no?

Reply:
In its current state it contains false information.

Reply:
Right, same thing. Reverted that bit.

Reply:
Right, it's often wrong, but if we clean up, then it should be cleaned up more properly, like in PR #13230? You weren't happy about that, and I do have some sympathy for trying and fixing the logging. I left it unchanged for now, as dropping pam_rhost is the part that we all seem to agree on.